composer.json

```diff
@@ -15,7 +15,7 @@
         "magento/magento-coding-standard": "^5.0"
     },
     "type": "magento2-module",
-    "version": "2.15.0",
+    "version": "2.16.0",
```

Comment:
If we bump version we should do it here as well https://github.com/BoltApp/bolt-magento2/blob/master/etc/module.xml#L21

Reply:
Updated!